docs/content/concepts/assets/multi-assets.mdx

```diff
@@ -28,109 +28,92 @@ The function responsible for computing the contents of any asset is an [op](/con
 
-If the `is_required` parameter is set to `False` on an output, and your function does not `yield` an <PyObject object="Output" /> object for that output, then no asset materialization event will be created, the I/O manager will not be invoked, downstream assets will not be materialized, and asset sensors monitoring the asset will not trigger.
+If the `skippable` parameter is set to `True` on an <PyObject object="AssetSpec" />, and your function does not `yield` a <PyObject object="MaterializeResult" /> object for that asset, then no asset materialization event will be created, downstream assets in the same run will not be materialized, and asset sensors monitoring the asset will not trigger.
 
 ```python file=/concepts/assets/multi_asset_conditional_materialization.py
 import random
 
-from dagster import AssetOut, Output, asset, multi_asset
+from dagster import AssetSpec, MaterializeResult, asset, multi_asset
 
 
 @multi_asset(
-    outs={"asset1": AssetOut(is_required=False), "asset2": AssetOut(is_required=False)}
+    specs=[AssetSpec("asset1", skippable=True), AssetSpec("asset2", skippable=True)]
 )
 def assets_1_and_2():
     if random.randint(1, 10) < 5:
-        yield Output([1, 2, 3, 4], output_name="asset1")
+        yield MaterializeResult(asset_key="asset1")
 
     if random.randint(1, 10) < 5:
-        yield Output([6, 7, 8, 9], output_name="asset2")
+        yield MaterializeResult(asset_key="asset2")
 
 
-@asset
-def downstream1(asset1):
-    # will not run when assets_1_and_2 doesn't materialize the asset1
-    return asset1 + [5]
+@asset(deps=["asset1"])
+def downstream1():
+    """Will not run when assets_1_and_2 doesn't materialize asset1."""
 
 
 @asset
 def downstream2(asset2):
-    # will not run when assets_1_and_2 doesn't materialize the asset2
-    return asset2 + [10]
+    """Will not run when assets_1_and_2 doesn't materialize asset2."""
 ```
 
 ### Subsetting multi-assets
 
 By default, it is assumed that the computation inside of a multi-asset will always produce the contents all of the associated assets. This means that attempting to execute a set of assets that produces some, but not all, of the assets defined by a given multi-asset will result in an error.
 
-Sometimes, the underlying computation is sufficiently flexible to allow for computing an arbitrary subset of the assets associated with it. In these cases, set the `is_required` attribute of the outputs to `False`, and set the `can_subset` parameter of the decorator to `True`.
+Sometimes, the underlying computation is sufficiently flexible to allow for computing an arbitrary subset of the assets associated with it. In these cases, set the `skippable` attribute of the asset specs to `True`, and set the `can_subset` parameter of the decorator to `True`.
```

Comment:
```suggestion
Sometimes, the underlying computation is sufficiently flexible to allow for computing an arbitrary subset of the assets associated with it. In these cases, you should set the following to `True`:

- The asset spec's `skippable` attribute
- The decorator's `can_subset` parameter
```

Similar suggestion here to make this less dense. I'm not sure I formatted 'spec' (or 'specs'?) here correctly, but overall, wdyt of this approach?

Reply:
Mind if I punt this suggestion to a separate change? Seems like a reasonable change to make, but out of scope of what I'm trying to fix in this PR.